sunpy/util/scraper.py

```diff
@@ -0,0 +1,198 @@
+import datetime
```

Comment:
if it uses #1002 it can be here. It will just need rebasing after #1002 is merged.